modules/installation-guide/partials/proc_configuring-the-number-of-workspaces-a-user-can-run-using-configmap.adoc

```diff
@@ -0,0 +1,34 @@
+[id="configuring-the-number-of-workspaces-a-user-can-run-using-configmap_{context}"]
+= Configuring the number of workspaces a user can run using ConfigMap
+
+This procedure describes how to run more than one workspace simultaneously using ConfigMap. 
+
+.Prerequisites
+* An instance of `{prod-short}` installed using ConfigMap.
+* The combination of PVC strategy and access mode meets the following criteria:
++
+--
```

Comment:
why do we need mentions of `PVC strategy`?

Reply:
We need to avoid that situation: `ReadWriteOnce` access mode  + `common` PVC strategy.